README.md

```diff
@@ -178,405 +128,133 @@ After installing the library you will need to require it in your test file (see
+That method is the opposite of the previous one, so, it waits for an element not to be visible in the page. By saying not being displayed means that the element may be in the DOM, but not visible.
+[Example](docs/EXAMPLES.md#waitforelementnottobevisible)
 
-        browser.get(homePageUrl);
+### `clickWhenClickable`
 
-        const goToContactPageButton = element(by.className("contact-button"));
+That method is used to click in an element only as soon as it is in a clickable state. This means that the element is visible and enabled for clicking.
+[Example](docs/EXAMPLES.md#clickwhenclickable)
 
-        goToContactPageButton.click();
+### `fillFieldWithTextWhenVisible`
 
-        protractorHelper.waitForUrlNotToBeEqualToExpectedUrl(homePageUrl, "URL is equal to " + homePageUrl, 3000);
+That method fills an input field with a text as soon as such field is visible.
+[Example](docs/EXAMPLES.md#fillfieldwithtextwhenvisible)
 
-        // ...
-    });
-});
-```
+### `fillInputFieldWithFileWhenPresent`
 
-### Example of usage of `waitForUrlToContainString`
+That method fills a file input field with a specified file as soon as the file input field is present in the DOM.
+[Example](docs/EXAMPLES.md#fillinputfieldwithfilewhenpresent)
 
-```js
-const protractorHelper = require("protractor-helper");
+### `clearFieldWhenVisible`
 
-describe("foo", () => {
-    it("bar", () => {
-        browser.get("https://example.com");
+That method clears a text input field as soon as such field is visible.
 
-        const goToContactPageButton = element(by.className("contact-button"));
+### `clearFieldWhenVisibleAndFillItWithText`
 
-        goToContactPageButton.click();
+That method clears a text input field as soon as such field is visible, and then it fills it with a text.
+[Example](docs/EXAMPLES.md#clearfieldwhenvisibleandfillitwithtext)
 
-        protractorHelper.waitForUrlToContainString("contact", "URL does not contains the string 'contact'", 3000);
+### `tapWhenTappable`
 
-        // ...
-    });
-});
-```
+That method performs a tap action on a clickable/tappable HTML element as soon is it is clickable/tappable. This method is used when performing web mobile testing in mobile emulators, for example.
+[Example](docs/EXAMPLES.md#tapwhentappable)
 
-### Example of usage of `waitForUrlNotToContainString`
+### `waitForTextToBePresentInElement`
 
-```js
-const protractorHelper = require("protractor-helper");
+That method waits for a specific text to be present in a specific HTML element.
+[Example](docs/EXAMPLES.md#waitfortexttobepresentinelement)
 
-describe("foo", () => {
-    it("bar", () => {
-        browser.get("https://example.com");
+### `waitForTextNotToBePresentInElement`
 
-        protractorHelper.waitForUrlNotToContainString("foobarbaz", "URL contains the string 'foobarbaz'", 3000);
+That method is the opposite of the previous one, so, it waits for a specific text not to be present in a specific HTML element.
```

Comment:
Same as others.

Reply:
Commit made because this review: ce8ef21